src/status_im2/contexts/chat/bottom_sheet_composer/sub_view.cljs

```diff
@@ -0,0 +1,33 @@
+(ns status-im2.contexts.chat.bottom-sheet-composer.sub-view
+  (:require
+    [react-native.blur :as blur]
+    [react-native.core :as rn]
+    [react-native.linear-gradient :as linear-gradient]
+    [react-native.reanimated :as reanimated]
+    [status-im2.contexts.chat.bottom-sheet-composer.style :as style]))
+
+(defn bar
+  []
+  [rn/view {:style style/bar-container}
+   [rn/view {:style (style/bar)}]])
+
+(defn blur-view
+  [layout-height]
+  [:f>
+   (fn []
+     [reanimated/view {:style (style/blur-container layout-height)}
+      [blur/view (style/blur-view)]])])
```

Comment:
Regarding https://github.com/status-im/status-mobile/pull/15639/files#r1168033834

I think it also makes you break our rule [of always declaring styles in :style key](https://github.com/status-im/status-mobile/blob/0f8aec00f26f0d32b67bf941270a3183671cf3fe/doc/new-guidelines.md#L64)

Reply:
Technically it is still declared in a `style` key. You can find a similar instance here `status-im2.contexts.activity-center.view`